### DISCUSSION TITLE
Mongo tables not updating

### DISCUSSION THREAD

**Discussion by bfg-coding** [prisma/prisma#19108] [Q&A] [answered]
### Bug description

I'm working on a personal project using Prisma and Mongodb I went and updated the schema to add some fields and table then performed a `pnpm dlx prisma db push`  and the table never updated.  I nuked the docker container running mongodb and tried again but now prisma is creating the table but not with any of the fields I'm specifying it's just generating the id field.   I've reverted to the original test build I did and same thing 

```prisma
datasource db {
  provider = "mongodb"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "prisma-client-js"
}

model User {
    id     String   @id @default(auto()) @map("_id") @db.ObjectId
    email  String   @unique
    name   String?
    birthYear  Int?
    createdAt  DateTime  @default(now())
    updatedAt  DateTime  @updatedAt
}
```

### How to reproduce

1. build schema file
2. run mongodb in container
3. run `pnpm dlx prisma db push` (npx should also perform the same thing)
4. witness bug (unless I'm just unlucky)

### Expected behavior

update table to match schema

### Prisma information

Prisma response to command 

```
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": MongoDB database "test" at "localhost:27017"

The database is already in sync with the Prisma schema.
```

### Environment & setup

- OS: PopOs (linux)
- Database: MongoDB
- Node.js version: 18.14.0

### Prisma Version

both 4.11 and 4.13

**Comment by nurul3101** [comment]
Hello @bfg-coding 👋 

Would it be possible for you to provide a minimal reproduction so that we can investigate and have a look?
Are you using MongoDB Atlas? Did you get any error message when you invoked the `db push` command?

**Reply by nurul3101** [reply]
@bfg-coding I am checking to see if you had a chance to look at my last response. Do let us know in case you need any assistance.

**Reply by bfg-coding** [reply]
Sorry I must have missed the notification on this. I'll try to get a minimal reproduction up tonight

**Reply by bfg-coding** [reply]
To answer your question I have done both docker run of Mongo and an Atlas run with the same issue or at least similar issue

**Reply by bfg-coding** [reply]
This is related but not the same but [TurboRepo](https://github.com/vercel/turbo/issues/4826) is the tool I'm using on the project so there is a strong chance that this issue is more on the their end then Prisma's but I wanted to make sure I checked both paths on this issue

**Reply by nurul3101** [reply]
A minimal reproduction would indeed be helpful.

**Reply by bfg-coding** [reply, marked as answer]
Finally had time to make a minimal reproduction and it worked even though I'm copying the same flow.  There must be something wonky with my setup.  I'm good to close this discussion, as the source of the issue is going to be deeper than what was mentioned here.  Thank you for your time and patience

**Reply by janpio** [reply]
Let us know when you figure out what was going wrong for you, others might benefit from it as well. Thanks and good luck!

**Reply by vpaul18** [reply]
@bfg-coding @janpio   Hey! I'm experiencing the same issues (basic NextJS setup). I used to be able to update it only after deleting `node_modules`, `npm i`, `npm i @prisma/client`, and then `npx prisma db push` (which is also very annoying).

But now it doesn't seem to update anymore (also tried `npx prisma generate`).
